ads/google/a4a/performance.js

```diff
@@ -250,4 +250,12 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
     this.element_.parentNode.insertBefore(pingElement, this.element_);
     dev().info('PING', url);
   }
+
+  /**
+   * Returns the initialization time of this reporter.
+   * @return {number} The initialization time in ms.
+   */
+  getInitTime() {
+    return this.initTime_;
+  }
```

Comment:
If you expose `getDeltaTime_` as a public method as well, then you can call it inside `extractAmpAnalyticsConfig` and skip doing the delta logic there yourself.  (Probably move all the initTime and getDeltaTime logic to BaseLifecycleReporter so that we can be agnostic about which subclass we're getting.)